### ISSUE TITLE
Right to left not support in Odoo CE 13 using Odoo services

### ISSUE THREAD

**Issue by imjaypatel** [odoo/odoo#42575]
Impacted versions: v13

Hi,
I am facing issue in RTL. In my server when I running Odoo manually RTL working fine. However, when I running Odoo using service. RTL not working. Any specially configuration required in `sudo nano /etc/odoo-server.conf` file?

Here is my `/etc/odoo-server.conf` file contains,

> [options]
> ; This is the password that allows database operations:
> admin_passwd = admin
> xmlrpc_port = 8069
> logfile = /var/log/odoo/odoo-server.log
> addons_path=/odoo/odoo-server/addons,/odoo/custom/addons

Thank you.

**Comment by obayit** [comment]
how did you install rtlcss? (It is mentioned [here](https://www.odoo.com/documentation/12.0/setup/install.html#dependencies))
maybe it is not installed for the user running odoo from the service!

**Comment by imjaypatel** [comment]
@obayit yes i installed it. Its work fine when running server manually. But it would not work when I running Odoo using service.

**Comment by Yenthe666** [comment]
Perhaps you're experiencing something very similar to this: https://github.com/Yenthe666/InstallScript/commit/e2693d5d5c20edca213eb4686e362e4eb14b10c3
